.eslintrc

```diff
@@ -20,9 +20,13 @@
     "sourceType": "module"
   },
   "globals": {
-    "AMP": false,
-    "context": false,
-    "global": false
+    "ANALYTICS_VENDOR_SPLIT": "readonly",
+    "NATIVE_CUSTOM_ELEMENTS_V1": "readonly",
+    "_RTVEXP_INABOX_LITE": "readonly",
```

Comment:
Can you move these to `build-system/global-configs/.eslintrc`?

Reply:
I don't know what that is.

Reply:
`.eslintrc` files are hierarchical. Just add a new one under `build-system/global-configs/`

![image](https://user-images.githubusercontent.com/26553114/67439004-11593e80-f5c3-11e9-99f6-011e375a8f7b.png)

Reply:
```
src/friendly-iframe-embed.js
  862:5  error  'NATIVE_CUSTOM_ELEMENTS_V1' is not defined  no-undef
```

I'm gonna leave this for you to do.

Reply:
Oh, I see. The strings are used in actual code as well.

> I'm gonna leave this for you to do.

SGTM.